src/interfaces.ts

```diff
@@ -149,24 +152,18 @@ export interface Cement {
 
 export interface MDPoint {
   point: Point;
+  normal?: Vector2;
   md: number; // Currently calculated MD
 }
 
-export interface HoleObjectData {
-  data: HoleSize;
-  points: MDPoint[];
-  hasShoe?: boolean;
-  innerDiameter?: number;
-}
-
 export interface WellItemGraphics {
   graphics: Graphics;
 }
 
 export interface NormalCoordsObject {
```

Comment:
IT looks like this interface is not used anymore after your changes, should we just get rid of it?